target_finder/src/nodes/target_locator.cpp

```diff
@@ -111,12 +117,13 @@ bool TargetLocatorService::executeCallBack( target_locater::Request &req, target
   ros::NodeHandle nh;
   CameraObservations camera_observations;
 
-  ROSCameraObserver camera_observer(image_topic_);
+  ROSCameraObserver camera_observer(image_topic_, camera_name_);
 
   // get the focal length and optical center
   double fx,fy,cx,cy; 
   double k1,k2,k3,p1,p2;// unused 
-  if(!camera_observer.pullCameraInfo(fx, fy, cx, cy, k1, k2, k3, p1, p2)){
+  int height, width; //unused
+  if(!camera_observer.pullCameraInfo(fx, fy, cx, cy, k1, k2, k3, p1, p2, width, height)){
     ROS_ERROR("could not access camera info");
   }
   camera_observer.clearObservations();
```

Comment:
@akgoins Take a look at my comments, make minor fixes, then ping me and I'll accept.